src/backend/src/controllers/users.controllers.ts

```diff
@@ -104,3 +105,45 @@ export const logUserIn = async (req: any, res: any) => {
 
   return res.status(200).json(authenticatedUserTransformer(user));
 };
+export const updateUserRole = async (req: any, res: any) => {
+  const updatingUserId: number = parseInt(req.params.userId);
```

Comment:
i think `targetUserId` is clearer because updating could mean the one doing the updating or the one being updated. change this everywhere it shows up too pls